docs/001_develop/03_client-capabilities/005_grids/002_entity-manager/01_entity-manager-grid.mdx

```diff
@@ -0,0 +1,169 @@
+---
+title: 'Grid configuration'
+sidebar_label: 'Grid configuration'
+id: entity-manager-grid
+keywords: [grid configuration, entity-management, entity manager, table setup, customization, data grid, UI, frontend, grid options, user interface, data display, micro-front-ends]
+tags:
+  - entity management
+  - frontend
+  - micro-front-ends
+  - profile management
+  - ui
+  - user management
+  - web
+  - grid configuration
+  - entity manager
+  - table setup
+  - customization
+  - data grid
+  - grid options
+  - user interface
+  - data display
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+The Grid is a key component of the **Entity Management**, enabling you to display and manage data from back-end resources. It offers a flexible, structured layout where each row represents an entity, and each column represents an attribute of that entity. The grid can be easily configured to pull data from specified back-end resources and is automatically populated with relevant information.
+
+This section explains how to configure and customize the grid to suit your application's needs.
+
+## Examples 
+
+### Grid columns
+If you want to control the appearance of the grid, use the `columns` attribute. This enables you to supply an array of `ColDef[]` properties to configure the columns that are displayed, their width and many other properties; these are the same properties that are used by the [Grid Pro](../003_grid-pro/index.mdx) component.
+
+The example array below creates six columns, specifying the width of each one. 
+- The `field` is the field in the data source.
+- The `headerName` is the text displayed in the column heading.
+- Three of the fields use `cellStyle` to control the alignment of the values displayed. 
+- One field (whose column heading is "Status") also uses `cellStyle` to control the colour of the values displayed. 
+
+```typescript
+const columnDefs: ColDef[] = [
```

Comment:
can we have a kotlin section describing the table used here?

Reply:
[chore: addressed PR feedback PTL-1513](https://github.com/genesiscommunitysuccess/docs/pull/1951/commits/8e0a48c144037b54d8e8856d88f850cd48e063d9) - I added simple server configuration but in index.md for COUNTERPARTY table

Here we only have column configuration, it is focused only on client-side.
Later, when we unify the examples throughout the docs, we can change and improve them, but for now it seems to be enough.